apps/storybook/src/TransferList.stories.tsx

```diff
@@ -0,0 +1,242 @@
+/* eslint-disable react/jsx-key */
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  TransferList,
+  TransferListProps,
+  IconButton,
+} from '@itwin/itwinui-react';
+
+import {
+  SvgChevronLeft,
+  SvgChevronRight,
+  SvgChevronLeftDouble,
+  SvgChevronRightDouble,
+} from '@itwin/itwinui-icons-react';
+
+export default {
+  component: TransferList,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+  title: 'Core/TransferList',
+} as Meta<TransferListProps>;
+
+export const Basic: Story<TransferListProps> = (args) => {
+  const [data, setData] = React.useState([
+    { name: 'Item 1', active: false },
+    { name: 'Item 2', active: false },
+    { name: 'Item 3', active: false },
+    { name: 'Item 4', active: false },
+    { name: 'Item 5', active: false },
+    { name: 'Item 6', active: false },
+  ]);
+
+  return (
+    <TransferList {...args}>
+      <TransferList.Area>
+        <TransferList.List role={'listbox'}>
```

Comment:
Why are you adding role='listbox'? I think that is already added in TransferList?

Reply:
This has been fixed